npm/package.json

```diff
@@ -40,11 +40,12 @@
   "dependencies": {
     "@nationalarchives/file-information": "1.0.44",
     "@nationalarchives/tdr-generated-graphql": "1.0.75",
-    "apollo-boost": "^0.4.9",
+    "@apollo/client": "^3.3.21",
     "aws-sdk": "^2.946.0",
     "govuk-frontend": "^3.13.0",
     "graphql": "^14.7.0",
     "keycloak-js": "11.0.3",
+    "react": "^17.0.2",
```

Comment:
Does apollo-client depend on the whole of React? Does React get bundled into our main.js? 😬